### PR TITLE
Add SLA Configuration

### PR DESCRIPTION
**Pull request by Pilchie**

### CODE REVIEW

**Comment by Pilchie** [comment]
@anurse @dougbu

**Comment by Pilchie** [comment]
Fixes #23.

File: SlaOptions.cs [hunk 1]
```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace DependencyFlow
+{
+    public class SlaOptions
+    {
+        public Dictionary<string, Sla> Repositories { get; set;  }
```
Comment (analogrelay) [inline SlaOptions.cs]:
```suggestion
        public IDictionary<string, Sla>? Repositories { get; set;  }
```

nits: `IDictionary` for public properties and it's not initialized so it gets to be nullable ;). Or you could initialize it to an empty dictionary, or a dictionary containing only a hard-coded `[Default]` value.

**Review by analogrelay — APPROVED** [review]
LGTM

File: SlaOptions.cs (hunk 1 above)
Comment (Pilchie) [reply SlaOptions.cs]: Fixed

File: Pages/Incoming.cshtml.cs [hunk 2]
```diff
@@ -54,7 +57,7 @@ public async Task OnGet(int channelId, string owner, string repo)
 
                 incoming.Add(new IncomingRepo(
                     build,
-                    shortName: gitHubInfo?.Repo,
+                    shortName: gitHubInfo?.Repo ?? "",
```
Comment (dougbu) [inline Pages/Incoming.cshtml.cs:65]: `string.Empty` 😈

**Review by dougbu — APPROVED** [review]
Not familiar w/ the overall project but the changes look good given #23 goals

**Review by BrennanConroy — APPROVED** [review]

**Pull request merged**